### ISSUE TITLE
Visualiser ?

### ISSUE THREAD

**Issue by Zakwil** [inkle/ink#490]
Hi,
I don't know if it's exists, but do you have a knot visualiser?
In order to check the flow : http://outspaced.fightingfantasy.net/SVG_Flowcharts/CYOA001.svgz

**Comment by danielcestrella** [comment]
Won't be any graph visualizer until some user do it, creators of the project don't want it.

**Comment by y-lohse** [comment]
[I made this](https://yannick-lohse.fr/graphink/) as a quick POC. See if it fits your needs :)

**Comment by KumoKairo** [comment]
@y-lohse it doesn't seem to handle newer Ink versions. Console is red with that "Ink is too fresh" error. Do you have an opensource repo somewhere to update it through the sources?

**Comment by y-lohse** [comment]
Yep, it's here : https://github.com/y-lohse/graphink PR to update inkjs is welcome :)
